src/diffusers/schedulers/scheduling_ddim_inverse.py

```diff
@@ -124,18 +169,20 @@ def __init__(
         beta_schedule: str = "linear",
         trained_betas: Optional[Union[np.ndarray, List[float]]] = None,
         clip_sample: bool = True,
-        set_alpha_to_zero: bool = True,
+        set_alpha_to_one: bool = True,
```

Comment:
This is backwards breaking, why do this change?

Reply:
`set_alpha_to_zero` should be captured in `kwargs`, and retrieved to display the deprecation message.